kpi/filters.py

```diff
@@ -55,7 +63,37 @@ def filter_queryset(self, request, queryset, view):
         return queryset.filter(**fields)
 
 
-class AssetOrderingFilter(filters.OrderingFilter):
+class AssetOrganizationUsageFilter(filters.OrderingFilter, DeploymentFilter):
+
+    DEFAULT_USAGE_ORDERING_FIELDS = [
+        'name',
+        'submissions_total',
+        'submissions_this_billing_period',
+        'data_storage',
+        'transcript_minutes',
+        'translation_characters',
```

Comment:
These should probably be omitted until the `AssetUsageSerializer` can actually filter by them - or commented out with a short explanatory comment.